### PR TITLE
chore: smaller career updates

### PR DESCRIPTION
**Pull request by georgiee**
Today I describe our latest job position "Staff Software Engineer" and I wanted to make sure it's displayed appropriately. I found smaller issues around sorting, text ellipsis and the amount of positions we want to display on the landingpage.

I also introduced a manual sort field in Personio (again workaround through an additional description field). This allows me to place distinct positions and arbitrary positions on our website. Any positions without a sorting will be sorted by their creation date.

1. Preview (landingpage career list with a max of 3 items and text ellipsis)
https://satellytescommain21751-gkcareerupdate.gtsb.io/

2. Preview (career list with correct sorting)
https://satellytescommain21751-gkcareerupdate.gtsb.io/career/

The new position if you are interested:
https://satellytescommain21751-gkcareerupdate.gtsb.io/career/staff-software-engineer/

Note:
The changes in Personio also cause the main build to fail which I did not expect. This PR will fix that issue.
Screen Shot 2022-01-11 at 18 17 10

### CODE REVIEW

**Comment by georgiee** [comment]
The [npm colors incident hit us](https://www.bleepingcomputer.com/news/security/dev-corrupts-npm-libs-colors-and-faker-breaking-thousands-of-apps/) ?  😂 I guess our dependency update run while this was online and it's in our lock file now ? I will include a resolution which we probably can remove soon but it's a working fix for now.

Screen Shot 2022-01-11 at 18 11 16

File: src/assets/locales/en/translations.json [hunk 1]
```diff
@@ -8,9 +8,9 @@
   "main.career.kicker": "Career at Satellytes",
   "main.career.text": "We are looking for developers with passion! You don't have much work experience yet? No problem. You will learn everything you need to know. You are already an expert? You will get opportunities and challenges to grow further with your experienced colleagues. Take a look at our open positions. We look forward to receiving your application.",
   "main.career.button": "All open situations",
-  "main.blog.title": "Ping, Echo, Printf",
+  "main.blog.title": "Blog",
   "main.blog.kicker": "News about Satellytes",
-  "main.blog.text": "Find best practices, interesting bits of technology and news about Satellytes and the team",
+  "main.blog.text": "Find best practices, interesting bits of technology and news about Satellytes and the team.",
```
Comment (georgiee) [inline src/assets/locales/en/translations.json:13]: I sneaked in those changes which I found after checking the landingpage

File: src/pages/index.tsx [hunk 2]
```diff
@@ -62,7 +62,7 @@ export const IndexPageQuery = graphql`
         }
       }
     }
-    allSyPersonioJob(filter: { lang: { eq: $language } }) {
+    allSyPersonioJob(filter: { lang: { eq: $language } }, limit: 3) {
```
Comment (georgiee) [inline src/pages/index.tsx:65]: We don't want to show all positions on the start page. We have a button to go to the career page with all positions.

File: src/components/pages/landingpage/career.tsx [hunk 3]
```diff
@@ -30,7 +40,7 @@ export const Career = ({ positions }: CareerProps) => {
       <TeaserGrid>
         {positions.map((item) => (
           <Teaser key={item.id} title={item.name} linkTo={item.slug}>
-            {item.short}
+            {textEllipsis(item.short, 200)}
```
Comment (georgiee) [inline src/components/pages/landingpage/career.tsx:43]: We have some lengthy short descriptions so I decided to introduce a text ellipsis. I'm sure I could do this with CSS or even through graphql but this was the easiest way for now.

**Review by feedm3 — APPROVED** [review]
LGTM 👍

**Pull request merged**